tests/a_scripts/inference/test_convert_composer_to_hf.py

```diff
@@ -1396,7 +1410,10 @@ def test_convert_and_generate(
         'scripts/train/yamls/pretrain/testing.yaml',
         pytest.param(
             'scripts/train/yamls/pretrain/testing-moe.yaml',
-            marks=pytest.mark.gpu,
+            marks=[pytest.mark.gpu, pytest.mark.skipif(
```

Comment:
instead of skipping all the sparse moe tests can we just switch them to use the grouped impl?